lib/engines/content_block_manager/app/services/content_block_manager/delete_edition_service.rb

```diff
@@ -0,0 +1,23 @@
+module ContentBlockManager
+  class DeleteEditionService
+    def call(content_block_edition)
+      if content_block_edition.draft?
+        document = content_block_edition.document
+        document.with_lock do
+          content_block_edition.destroy!
+          if document_has_no_editions(document)
```

Comment:
Should this be `document_has_no_more_editions?` (I tend to prefer appending a `?` on the end of methods that return a boolean)

Reply:
good idea! renamed and added ?